template/public/app.js

```diff
@@ -1,34 +1,48 @@
-import moment from 'moment';
-import { uiModules } from 'ui/modules';
-import uiRoutes from 'ui/routes';
+import React from "react";
+import moment from "moment";
+import { uiModules } from "ui/modules";
+import chrome from "ui/chrome";
+import uiRoutes from "ui/routes";
+import { render, unmountComponentAtNode } from "react-dom";
 
-import 'ui/autoload/styles';
-import './less/main.less';
-import template from './templates/index.html';
+import "ui/autoload/styles";
+import "./less/main.less";
+import template from "./templates/index.html";
+import { Main } from "./components/main";
 
+const app = uiModules.get("apps/<%= name %>");
+app.config(stateManagementConfigProvider => {
+  stateManagementConfigProvider.disable();
+});
+const controller = ($scope, $route, $interval) => {
```

Comment:
Likewise, I'd be able to scan this function more easily with newlines on lines 20, 25, 26, and 30.